packages/cli/lib/projects.js

```diff
@@ -103,14 +112,45 @@ const validateProjectConfig = projectConfig => {
     process.exit(1);
   }
 
-  if (!fs.existsSync(projectConfig.srcDir)) {
+  if (!fs.existsSync(path.resolve(projectDir, projectConfig.srcDir))) {
     logger.error(
       `Project source directory '${projectConfig.srcDir}' does not exist.`
     );
     process.exit(1);
   }
 };
 
+const ensureProjectExists = async (accountId, projectName) => {
+  try {
+    await fetchProject(accountId, projectName);
+  } catch (err) {
+    if (err.statusCode === 404) {
```

Comment:
This may not be worth supporting, but what if we get a 500 or some other internal error code?

Reply:
My intent was for https://github.com/HubSpot/hubspot-cli/pull/564/files/370500c158ffbe7d34e32fdfef2c1ebdff146f1f#diff-fa4fa15b34bd2259fc2c24ec990fdbaa6492e6030d940700895318b399058174R150 to handle any other error codes that might come through